backend/user/middleware.go

```diff
@@ -159,6 +159,7 @@ func NewUserMiddleware(queries *sqlc.Queries, membersClient *members.Client) fun
 			span.AddEvent("User failed to load")
 			userCache.Set(userID, u, cache.WithExpiration(1*time.Minute))
 			ctx.Set(CtxUser, u)
+			return
```

Comment:
This should have been there all along. We bail out if there is an error getting person data, not related to this pr but it slipped in.